### ISSUE TITLE
No argument validation check.

### ISSUE THREAD

**Issue by wizzup** [svanderburg/node2nix#138]
It seem like `node2nix` don't check for invalid argument. 

This could be the source of confusing if there any type on argument, for example, `--deopment` instead of `--develpment`.

```
$ node2nix --version
node2nix 1.6.0

$ node2nix --whatever --developmento
fetching local directory: ./. from .
```

**Comment by svanderburg** [comment]
Agreed. I probably need to use a better option parsing library to make this happen. I'll investigate (hopefully) soon.

**Comment by svanderburg** [comment]
I just merged a pull request that seems to fix this: b562936d7861a6601cb0a680893ff5772aa5ca5b
